### PR TITLE
Move questions to a separate file

### PR DESCRIPTION
**Pull request by AjayPoshak**

### CODE REVIEW

**Comment by AjayPoshak** [comment]
I have started refactoring `prompts.js` file. Moving the questions to a separate file seemed obvious.

**Comment by sendilkumarn** [comment]
Do you intend to make it `ts` ? or can I merge this ?

**Comment by AjayPoshak** [comment]
Okay, I'll make it `ts` first.

**Comment by sendilkumarn** [comment]
Check with @rishabh3112 too, he was working on similar thingie :)

**Comment by sendilkumarn** [comment]
@AjayPoshak Can you resolve conflicts here? We can merge this.

**Comment by AjayPoshak** [comment]
I'll resolve it.

**Comment by sendilkumarn** [comment]
Thanks 👍

**Comment by AjayPoshak** [comment]
@rishabh3112 thanks, will do it.

@sendilkumarn hey, I am not getting the indentation. `editorconfig` says `tabs` whereas `ts` seems to expecting `spaces`.  Which one are we going ahead with?

**Comment by rishabh3112** [comment]
Yeah by our TSlint configuration we have space indentation. So, as this has been mutually committed by us in #66 . So, Go with spaces 👍.

**Comment by AjayPoshak** [comment]
@sendilkumarn @rishabh3112 Please review, I have done TS changes as well.

File: .editorconfig [hunk 1]
```diff
@@ -1,8 +1,8 @@
 root = true
 
 [*]
-indent_style = tab
-indent_size = 4
+indent_style = space
+indent_size = 2
```
Comment (sendilkumarn) [inline .editorconfig]: 😞

**Pull request merged**